YvesIraguha/lost-and-found-app
# chore(setup): setting up react project

#### What does this PR do?

- Setup react project 

#### Description of Task to be completed? 

- Create empty react project using create-react-app

#### How should this be manually tested?

- Clone the repo 

- In your home directory  cd this branch `ch-setting-react-project`

- Then run **`npm install`** to install all dependencies

- Run **`npm start`** to start the server

- In your default browser navigate to **`http://localhost:3000`**

> You should see a Rendered welcoming message

#### Any background context you want to provide?

- N/A
